course_content/iris_course/solutions/iris_exercise_6.1d

```diff
@@ -0,0 +1,14 @@
+
+# Iris course sample answer 6.1d
+# changing cube metadata items
+
+e1_copy = e1.copy()
```

Comment:
As far as I'm aware, we haven't introduced copy.
We also don't explain why its necessary.